src/scenes/Dashboard/ProgressReportsWidget/ProgressReportsExpandedGrid.tsx

```diff
@@ -96,7 +96,7 @@ export const ProgressReportsExpandedGrid = (props: Partial<DataGridProps>) => {
           [`.MuiDataGrid-row--dynamicHeight .MuiDataGrid-cell:not(.${ExpansionMarker})`]:
             {
               // Magic number to keep text vertically unchanged when the row is expanded
-              pt: '17.5px',
+              pt: '16px',
```

Comment:
16px is creating a small jump in lines that have no text. 16.5px seems to work better.

Reply:
16.5 causes a jump in Safari. 16 doesn't cause a jump for me in either...

Reply:
Ok I think I've got it 000e304
You were on the right track with the `line-height` - it just needed to be the full height (overriding `inherit` set by auto row height).